shared/oae/api/oae.api.authentication.js

```diff
@@ -237,7 +237,7 @@ define(['exports', 'jquery', 'oae.api.config', 'oae.api.i18n', 'oae.api.user', '
 
         // Use the `auth.html` `authExternalButton` macro to create a form that performs this
         // authentication
-        var $template = $('<div class="hide">${authExternalButton(strategy, opts)}</div>');
+        var $template = $('<div><!--${authExternalButton(strategy, opts)}--></div>');
```

Comment:
We can then remove the `addClass` and it'll make it easier to make changes to this template further down the line